coverage/PENDING_SUBMIT/amdvlk/0575-COVERAGE-cfg-282/reduced_manual/variant/shader.frag

```diff
@@ -0,0 +1,110 @@
+#version 320 es
+#define _int_1 _GLF_uniform_int_values[0]
+#define _int_4 _GLF_uniform_int_values[1]
+#define _int_35 _GLF_uniform_int_values[2]
+#define _int_0 _GLF_uniform_int_values[3]
+#define _float_1_0 _GLF_uniform_float_values[0]
+#define _float_0_0 _GLF_uniform_float_values[1]
+#define _float_2_0 _GLF_uniform_float_values[2]
+
+precision highp float;
+precision highp int;
+
+// Contents of _GLF_uniform_float_values: [1.0, 0.0, 2.0]
+layout(set = 0, binding = 0) uniform buf0
+{
+    float _GLF_uniform_float_values[3];
+};
+
+// Contents of _GLF_uniform_int_values: [1, 4, 35, 0]
+layout(set = 0, binding = 1) uniform buf1
+{
+    int _GLF_uniform_int_values[4];
+};
+
+const int _GLF_global_loop_bound = 100;
+int _GLF_global_loop_count = 0;
+
+struct S
+{
+    int data0;
+    int data1;
+};
+
+int a = 1;
+
+S arr[10] = S[10](S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1), S(1, 1));
+
+layout(location = 0) out vec4 _GLF_color;
+
+// Has no effect because the original value is always one.
```

Comment:
Also the call to this function is unreachable - could be worth noting.

Reply:
done